src/demo-app/a11y/dialog/dialog-neptune-iframe-a11y.html

```diff
@@ -0,0 +1,12 @@
+<h2 md-dialog-title>Neptune</h2>
```

Comment:
Adding a config option makes sense to me. 

I added another example for forms. The focus goes to the first input box in the form when the dialog is opened.